### PR TITLE
update ccall docstring

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

File: base/docs/basedocs.jl [hunk 1]
```diff
@@ -555,20 +555,20 @@ Short-circuiting boolean OR.
 kw"||"
 
 """
-    ccall((symbol, library) or function_pointer, ReturnType, (ArgumentType1, ...), ArgumentValue1, ...)
+    ccall((symbol, library) or function_pointer, returntype, (argtype1, ...), argvalue1, ...)
```
Comment (musm) [inline base/docs/basedocs.jl]: should probably change `(symbol, library)` to `(function_name, library)`

**Review by tkelman — APPROVED** [review]
thanks, I think this is an improvement and more in line with docstring conventions elsewhere

File: base/docs/basedocs.jl [hunk 2]
```diff
@@ -555,20 +555,20 @@ Short-circuiting boolean OR.
 kw"||"
 
 """
-    ccall((symbol, library) or function_pointer, ReturnType, (ArgumentType1, ...), ArgumentValue1, ...)
+    ccall((function_name, library) or function_pointer, returntype, (argtype1, ...), argvalue1, ...)
```
Comment (musm) [inline base/docs/basedocs.jl]: Should this be seperated into two?
```julia
ccall((function_name, library), returntype, (argtype1, ...), argvalue1, ...)

ccall(function_pointer, returntype, (argtype1, ...), argvalue1, ...)
```

**Review by Sacha0 — APPROVED** [review]
Thanks @musm! :)

**Review by Sacha0 — APPROVED** [review]
lgtm for touchups! :)

File: base/docs/basedocs.jl [hunk 3]
```diff
@@ -555,7 +555,9 @@ Short-circuiting boolean OR.
 kw"||"
 
 """
-    ccall((function_name, library) or function_pointer, returntype, (argtype1, ...), argvalue1, ...)
+    ccall((function_name, library), returntype, (argtype1, ...), argvalue1, ...)
+
```
Comment (tkelman) [inline base/docs/basedocs.jl]: I think other cases of multiple signatures in the same docstring don't have a blank line between them

**Pull request merged**